README.md

```diff
@@ -5,7 +5,7 @@ This is the public helm repository for Citizens Advice charts.
 ## Adding to Helm
 
 ```
-helm repo add citizensadvice 'https://raw.githubusercontent.com/citizensadvice/helm-charts/master/'
+helm repo add citizensadvice 'https://citizensadvice.github.io/helm-charts'
```

Comment:
if we change the repo url to the gh-pages one, we have to keep the old index.yaml in the master branch to avoid breaking compatibility.

Reply:
This is true, we do. I have no intention of removing the old index from master.